### DISCUSSION TITLE
TypeError: __init__() got an unexpected keyword argument 'check_same_thread'

### DISCUSSION THREAD

**Discussion by nerd-1** [sqlalchemy/sqlalchemy#8551] [Usage Questions]
When I connect the mysql database with sqlalchemy, it occurs this error, how can I do? My sqlalchemy version is 1.4.41, pymysql version is 1.0.2.

**Comment by CaselIT** [comment]
Hi,

Do you have `check_same_thread=True` in the url? Otherwise sqlalchemy does not passes it to mysql
